servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile_inc.glsl

```diff
@@ -14,6 +14,7 @@
 #endif
 #endif
 
+#define DRAW_CALL_PUSH_CONSTANT
```

Comment:
I would prefer naming this something like ``USING_MOBILE_RENDERER`` instead. There is a good chance we will stop using push constants for per-draw call information eventually, and then the define will quickly become out of date.

Reply:
ok